theorafile.c

```diff
@@ -127,15 +128,18 @@ int tf_open_callbacks(void *datasource, OggTheora_File *file, tf_callbacks io)
 
 
 	ogg_sync_init(&file->sync);
-	vorbis_info_init(&file->vinfo);
-	vorbis_comment_init(&file->vcomment);
 	th_info_init(&file->tinfo);
 	th_comment_init(&file->tcomment);
 
 	/* Is there even data for us to read...? */
 	TF_OPEN_ASSERT(INTERNAL_readOggData(file) <= 0)
 
 	/* Read header */
+	vorbis_info vinfo;
```

Comment:
The variables have to be at the top, for VS2010. The function calls can go up to where they used to be, to minimize the diff size.

Reply:
Didn't realize that was a style an issue in this century.  Updated.